nuvla/job_engine/connector/docker_compose.py

```diff
@@ -142,15 +142,24 @@ def stop(self, **kwargs):
         project_name = kwargs['name']
         docker_compose = kwargs['docker_compose']
         env = kwargs.get('env')
+        remove_images = kwargs.get('remove-images', False)
+        remove_volumes = kwargs.get('remove-volumes', False)
 
         with TemporaryDirectory() as tmp_dir_name:
             compose_file_path = f'{tmp_dir_name}/{docker_compose_filename}'
             compose_file = open(compose_file_path, 'w')
             compose_file.write(docker_compose)
             compose_file.close()
 
-            cmd = self.build_cmd_line(
-                ['-p', project_name, '-f', compose_file_path, 'down'])
+            cmd = ['-p', project_name, '-f', compose_file_path, 'down']
+
+            if remove_volumes:
+                cmd.append('--volumes')
+
+            if remove_images:
+                cmd.append('--rmi')
```

Comment:
I'm getting following error:
"RuntimeError: flag needs an argument: --rmi"

And if I check the command help it says the following:
--rmi string       Remove images used by services. "local" remove only images that don't have a custom tag ("local"|"all")

Reply:
Pushed a fix with rmi=all. If any objection scream

Reply:
All good. And I forgot it after the tests I did I detected the fail. Thank you for the fix.